x/protocolpool/keeper/msg_server.go

```diff
@@ -103,6 +103,14 @@ func (k MsgServer) CommunityPoolSpend(ctx context.Context, msg *types.MsgCommuni
 	return &types.MsgCommunityPoolSpendResponse{}, nil
 }
 
+func (k MsgServer) CreateContinuousFund(ctx context.Context, msg *types.MsgCreateContinuousFund) (*types.MsgCreateContinuousFundResponse, error) {
+	return &types.MsgCreateContinuousFundResponse{}, nil
+}
+
+func (k MsgServer) CancelContinuousFundProposal(ctx context.Context, msg *types.MsgCancelContinuousFundProposal) (*types.MsgCancelContinuousFundProposalResponse, error) {
+	return &types.MsgCancelContinuousFundProposalResponse{}, nil
```

Comment:
Build is probably gonna fail because this does not match the generated types, but other than that lgtm!

Reply:
oh right